balena/settings.py

```diff
@@ -199,8 +201,6 @@ def set(self, key, value):
 
     def remove(self, key):
         """
-        Remove a setting.
```

Comment:
Can't say if it's the best approach to remove the docstring description here altogether. I suppose your reasoning was that it is self-explanatory?

Reply:
This was a miss deletion :)